database_admin/migrations/058_mark_stale_fix.up.sql

```diff
@@ -0,0 +1,92 @@
+CREATE OR REPLACE FUNCTION delete_systems(inventory_ids UUID[])
```

Comment:
You should explicitly remove old `delete_systems(ids INT[])`, as you define a new function.